message MDropData
{
	optional int32		type = 1;
	optional int32		index = 2;
	optional int32		amount = 3;
}

message MDropList
{
	repeated MDropData		drops = 1;
}